### PR TITLE
2 Hospital Program Pages

### PR DESCRIPTION
**Pull request by innocentmm**
Only uploading the pages that have info

### CODE REVIEW

File: frontend/pages/royalPerthHospital.js [hunk 1]
```diff
@@ -17,36 +16,92 @@ class RoyalPerthHospital extends Component {
...
+                  alt="RPH Work"
+                  style={{
+                    marginTop: '15px',
+                    marginBottom: 0,
+                    width: '1080x',
+                    height: '220px',
+                    color: '#424242' // offblack
+                  }}
+                  src="/images/hospitalPrograms/royalPerth/003_Homeless-Healthcare_160914.png"
+                />
```
Comment (noobling) [inline frontend/pages/royalPerthHospital.js:58]: Please remove commented out block

File: frontend/pages/royalPerthHospital.js [hunk 2]
```diff
@@ -17,36 +16,92 @@ class RoyalPerthHospital extends Component {
...
+                <div className="col left">
+                  <img
+                    alt="RPH Image"
+                    src="/images/hospitalPrograms/royalPerth/003_Homeless-Healthcare_160914.png"
+                    style={{ height: 400 }}
+                  ></img>
+                  <ul className="col right">
+                    <h4>Further Reading</h4>
+                    <li>
+                      <a href="https://rph.health.wa.gov.au/About-us/News/RPHs-Homeless-Team-is-a-WA-Health-Excellence-Awards-finalist">
```
Comment (noobling) [inline frontend/pages/royalPerthHospital.js:72]: for all your external links I would recommend using `target="_blank"` `ref="noopener noreferrer"` to open in a new tab

File: frontend/pages/royalPerthHospital.js [hunk 3]
```diff
@@ -17,36 +16,92 @@ class RoyalPerthHospital extends Component {
...
+          <a className="carousel-item" href="#three!">
+            <img src="https://lorempixel.com/250/250/nature/3" />
+          </a>
+          <a className="carousel-item" href="#four!">
+            <img src="https://lorempixel.com/250/250/nature/4" />
+          </a>
+          <a className="carousel-item" href="#five!">
+            <img src="https://lorempixel.com/250/250/nature/5" />
+          </a>
+        </div> */}
```
Comment (noobling) [inline frontend/pages/royalPerthHospital.js:104]: Please remove commented code

File: frontend/pages/royalPerthHospital.js (hunk 2 above)
Comment (innocentmm) [reply frontend/pages/royalPerthHospital.js:72]: okay will be done

File: frontend/pages/royalPerthHospital.js (hunk 3 above)
Comment (innocentmm) [reply frontend/pages/royalPerthHospital.js:104]: Done

**Comment by innocentmm** [comment]
I think this should be all good

**Pull request merged**